src/Makefile.am

```diff
@@ -493,6 +498,18 @@ libzcash_a_LDFLAGS = $(HARDENED_LDFLAGS)
 
 libzcash_a_CPPFLAGS += -DMONTGOMERY_OUTPUT
 
+# libsnark
+libsnark_a_SOURCES = \
+  libsnark/algebra/curves/alt_bn128/alt_bn128_g1.cpp \
+  libsnark/algebra/curves/alt_bn128/alt_bn128_g2.cpp \
+  libsnark/algebra/curves/alt_bn128/alt_bn128_init.cpp \
+  libsnark/algebra/curves/alt_bn128/alt_bn128_pairing.cpp \
+  libsnark/algebra/curves/alt_bn128/alt_bn128_pp.cpp \
+  libsnark/common/utils.cpp \
+  libsnark/common/profiling.cpp
+
+libsnark_a_CPPFLAGS = -fPIC -O2 -march=x86-64 -DMONTGOMERY_OUTPUT -DBINARY_OUTPUT -DNO_PT_COMPRESSION=1 -DMULTICORE -DCURVE_ALT_BN128 -fopenmp -DNO_PROCPS -static -DSTATIC -I./libsnark/
```

Comment:
`NO_DOCS` and `NO_SUPERCOP` are no longer relevant with our fork and build infrastructure.

We do actually pass `-march=x86-64` currently in the depends system, but I don't know why.

And finally nah it doesn't matter as far as I know.